torch/_dynamo/output_graph.py

```diff
@@ -73,6 +73,21 @@ class OutputGraphState(NamedTuple):
     nn_modules: Optional[Dict[str, torch.nn.Module]]
     side_effects: SideEffects
     timestamp: int
+    name_to_input: OrderedDict[str, Optional[fx.Proxy]]
```

Comment:
Where does this get written to or read from?

Reply:
uhhh, just grep for `name_to_input` in this file? I don't really understand the Q

Reply:
it's used for picking input names for the graph. seemed like it should be saved.